src/main/kotlin/org/ionproject/integration/utils/StringUtils.kt

```diff
@@ -0,0 +1,22 @@
+package org.ionproject.integration.utils
+
+import org.ionproject.integration.model.external.timetable.Language
+
+object IgnoredWords {
+    private val portugueseWords = listOf("de", "do", "da", "e", "o", "a", "para", "dos", "das")
```

Comment:
I would add "Dr." "Dra." "Dr" "Dra"
Example: Escola Superior de Saúde Dr. Lopes Dias

Reply:
I'll add that this school has a particular "strange" acronym, since that in their website the acronym is as ESALD.

For the D. as in D. Henrique, I've seen in the search that is used in the acronym, like Escola Superior Náutica Infante D. Henrique which is in their website as ENIDH.

Reply:
Done